### DISCUSSION TITLE
Allow to Un-hide posts

### DISCUSSION THREAD

**Discussion by lorthirk** [dailydotdev/daily#1352] [Feature Requests]
Hello,

This morning I was looking at the Private Guild feature, and I hid a couple of posts to play around. However, I noticed that I cannot un-hide them anymore, which I think it's a bad experience. Is it possible to implement a list of hidden posts somewhere, so that we can recover them and un-hide if needed?

**Comment by idoshamun** [comment]
Hi @lorthirk,

Thanks for the suggestion. It's a great papercut to fix
